frontend/amundsen_application/static/js/features/ColumnList/ColumnType/parser.ts

```diff
@@ -191,32 +208,29 @@ export function convertNestedTypeToColumns(
     if (typeof child === 'string') {
       const [columnName, colType] = child.split(COLUMN_TYPE_SEPARATOR);
       if (colType !== undefined) {
-        nestedColumns.push({
-          badges: [],
-          col_type: colType.replace(SEPARATOR_DELIMETER, ''),
-          description: '',
-          name: columnName,
-          sort_order: 0,
-          nested_level: nestedLevel,
-          is_editable: false,
-          stats: [],
-        });
+        const column = createNestedColumn(
+          columnName,
+          colType.replace(SEPARATOR_DELIMETER, ''),
+          nestedLevel
+        );
+        nestedColumns.push(column);
       }
     } else {
+      // Changing nestedLevel introduces a scope-level bug.
+      let incNestedLevel = 0;
```

Comment:
What does "inc" stand for? Increment?
I would not abbreviate it.

Reply:
Updated